packages/teams-js/src/public/meeting.ts

```diff
@@ -157,12 +157,23 @@ export namespace meeting {
     isSpeakingDetected: boolean;
   }
 
+  export interface MeetingReactionReceivedEventData {
+    /**
+     * Indicates the type of MeetingReaction received
+     */
+    meetingReactionType?: MeetingReactionType;
```

Comment:
Should this property be called `reactionType`? We're already in a `MeetingReactionReceivedEventData` type, and the type of the enum is `MeetingReactionType`, so we know it's about meeting reactions. I don't know. Seems a bit redundant.

Reply:
In Teams codebase we have to be specific about the type of reactions because meeting reactions are different than chat reactions. I think keeping meetingReactionType is the best way to go.

Reply:
sounds good.